src/components/NcActionInput/NcActionInput.vue

```diff
@@ -171,6 +190,22 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		/**
+		 * The hidden input label for accessibility purposes.
+		 */
+		label: {
+			type: String,
+			default: undefined,
+		},
+		/**
+		 * We normally have the label hidden visually and use it for
+		 * accessibility only. If you want to have the label visible just above
+		 * the input field pass in true to this prop.
```

Comment:
from my understanding, I thought we always wanted the labels visible

this is also why we have made them visible

so defaulting to false would be just for compatibility?

Reply:
Default was decided in https://github.com/nextcloud/nextcloud-vue/pull/2868, perhaps @marcoambrosini could detail the reasoning behind this?

Login page explicitly opts-in to using the visible label https://github.com/nextcloud/server/blob/be0facaba478235ee5f0608e36bf806570298a18/core/src/components/login/LoginForm.vue#L59-L73

Reply:
I think that's what I first did but I then reverted for backwards compatibility yes after discussing it. It's totally fine for me to have it visible by default

Reply:
Defaulted `labelVisible` to `true`, `NcInputField` would have to stay hidden by default for v7 at least